README.md

```diff
@@ -1 +1,66 @@
-# awesome-book-es6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> awesome-books
```

Comment:
- [ OPTIONAL ] I suggest adding some information about the project and more description to describe what this app for?